tensorflow/core/kernels/segment_reduction_ops_gpu_0.cu.cc

```diff
@@ -33,36 +49,32 @@ bool DisableSegmentReductionOpDeterminismExceptions() {
 
 namespace functor {
 
-#define DEFINE_SORTED_GPU_SPECS_INDEX(T, Index)                           \
-  template struct SegmentReductionFunctor<T, Index, functor::Zero<T>,     \
-                                          functor::NonAtomicSumOpGpu<T>,  \
-                                          functor::AtomicSumOpGpu<T>>;    \
-  template struct SegmentReductionFunctor<T, Index, functor::One<T>,      \
-                                          functor::NonAtomicProdOpGpu<T>, \
-                                          functor::AtomicProdOpGpu<T>>;   \
-  template struct SegmentReductionFunctor<T, Index, functor::Highest<T>,  \
-                                          functor::NonAtomicMinOpGpu<T>,  \
-                                          functor::AtomicMinOpGpu<T>>;    \
-  template struct SegmentReductionFunctor<T, Index, functor::Lowest<T>,   \
-                                          functor::NonAtomicMaxOpGpu<T>,  \
-                                          functor::AtomicMaxOpGpu<T>>;
+#define DEFINE_SORTED_GPU_SPECS_INDEX(T, Index)                            \
+  template struct SegmentReductionFunctor<T, Index, functor::Zero<T>,      \
```

Comment:
Can you please add `/*EmptySegmentValueF=*/` and `/*InitialValueF=*/` here to make it clear which arg is which?    Same elsewhere.

Reply:
Done